deploy/kubernetes/helm/che/templates/configmap.yaml

```diff
@@ -92,6 +92,7 @@ data:
   CHE_WORKSPACE_HTTPS__PROXY: {{ .Values.cheWorkspaceHttpsProxy | quote}}
   CHE_WORKSPACE_NO__PROXY: {{ .Values.cheWorkspaceNoProxy | quote}}
   CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT: '{{ .Values.global.workspaceIdleTimeout }}'
+  CHE_LIMITS_USER_WORKSPACES_RUN_COUNT: {{ .Values.global.workspace.number | quote }}
```

Comment:
I know the documentation says we should use `global.workspace.number, but I'm debating if this should be a `global` setting at all?  It is only use in the che chart if I'm not mistaken.
It would make more sense to put it here: `che.workspace.number`

Reply:
Could you point to the documentation in question?

Reply:
https://www.eclipse.org/che/docs/che-7/installation-guide/running-more-than-one-workspace-at-a-time/

Reply:
The doc exists but functionally doesn't ;)
We can update the doc it means we can find better names

Reply:
I'm game to make those changes. 😃 Just let me know what you would like to see (che.workspace.number?  Something else?)

Reply:
In case you haven't seen it, I also have a PR opened for `che-docs`: https://github.com/eclipse/che-docs/pull/1709